play-services-nearby-core/src/main/kotlin/org/microg/gms/nearby/exposurenotification/DeviceInfo.kt

```diff
@@ -13,31 +13,31 @@ import android.util.Log
 import com.google.android.gms.nearby.exposurenotification.CalibrationConfidence
 import kotlin.math.roundToInt
 
-data class DeviceInfo(val oem: String, val model: String, val txPowerCorrection: Byte, val rssiCorrection: Byte, @CalibrationConfidence val confidence: Int = CalibrationConfidence.MEDIUM)
+data class DeviceInfo(val oem: String, val device: String, val model: String, val rssiCorrection: Byte, val txPowerCorrection: Byte, @CalibrationConfidence val confidence: Int = CalibrationConfidence.MEDIUM)
```

Comment:
Beside adding `device`, I inverted `rssiCorrection` and `txPowerCorrection` according to my understanding of the CSV file header.